packages/diagnostic/.eslintrc.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-undef */
```

Comment:
confirm this aligns with the latest PR's which "optimize" various configs and consume from global

Reply:
This is "default" config copied from other packages. I will optimize linter configs later.